crates/bevy_text/src/text.rs

```diff
@@ -167,6 +169,9 @@ pub struct TextStyle {
 impl Default for TextStyle {
     fn default() -> Self {
         Self {
+            #[cfg(feature = "default_font")]
```

Comment:
If we want user-defined default fonts to be supported without the FiraMono default font, this should point to the default font handle unconditionally right?

Reply:
forgot this one 👍 

also it's not really needed, but better for consistency